fdbclient/NativeAPI.actor.h

```diff
@@ -418,7 +416,7 @@ class Transaction : NonCopyable {
 	Database cx;
 
 	double backoff;
-	Version committedVersion;
+	Version committedVersion{ invalidVersion };
```

Comment:
I mentioned this offline, but it's possible someone was relying on the property that getting the committed version returned `invalidVersion` before commit finished. In this current version, that may not happen if the transaction hasn't been initialized yet. I don't know if this is something we can guard with API versioning, but that might be the safest approach.

Reply:
This will now be initialized in the `ThreadSafeTransaction` constructor, so if a client calls `ThreadSafeTransaction::getCommittedVersion` before the commit is complete, they are still guaranteed to receive `::invalidVersion`.

Reply:
Oh you're right, nevermind then.